controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -270,15 +272,28 @@ func (r *StorageClusterReconciler) reconcileUninstallAnnotations(sc *ocsv1.Stora
 	return false, nil
 }
 
-// verifyNoStorageConsumerExist verifies there are no storageConsumers on the same namespace
-func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.StorageCluster) error {
+// verifyNonLocalStorageConsumerExist verifies there are no storageConsumers on the same namespace
+func (r *StorageClusterReconciler) verifyNonLocalStorageConsumerExist(instance *ocsv1.StorageCluster) error {
```

Comment:
Semantically, you're changing the meaning of the function name. Use something like `verifyNoRemoteStorageConsumersExist()`.

Reply:
I had to change function name to match updated function body 😅.